bool send_true
int8 deviceID
int8[] reqdata
---
int16[] resdata
